Configure labeler for 'module: distributed'

wconstab opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #112813
* __->__ #112812

To opt-in to getting notified based on this label, simply add yourself
to the summary field at the top of this issue:
https://github.com/pytorch/pytorch/issues/24422

Uses same file paths as current CODEOWNERS.

Note: can easily add sub-labels for components within distributed if we
want.

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @yf225 @kiukchung @d4l3k

wconstab committed:
Configure labeler for 'module: distributed'

To opt-in to getting notified based on this label, simply add yourself
to the summary field at the top of this issue:
https://github.com/pytorch/pytorch/issues/24422

Uses same file paths as current CODEOWNERS.

Note: can easily add sub-labels for components within distributed if we
want.

[ghstack-poisoned]

wanchaol reviewed (approved):
lgtm, but there's no module:distributed in that issue yet? Maybe we should add one and remove the `oncall:distributed`

fduwjj reviewed (approved):
Just created a label "module:distributed". Btw, if someone does not like its color, feel free to change, it. lol

wconstab commented:
Note: I am not sure if the space matters but the label I added to the issue was module: distributed and it looks like @fduwjj created it without the space. May have to reconcile.

wconstab committed:
Update on "Configure labeler for 'module: distributed'"


To opt-in to getting notified based on this label, simply add yourself
to the summary field at the top of this issue:
https://github.com/pytorch/pytorch/issues/24422

Uses same file paths as current CODEOWNERS.

Note: can easily add sub-labels for components within distributed if we
want.

[ghstack-poisoned]

fduwjj commented:
@wconstab Wait, the lable I created does come with the tab. you can check it here: https://github.com/pytorch/pytorch/labels?q=distribute

wconstab commented:
@pytorchbot merge

wz337 commented on .github/labeler.yml:
```diff
@@ -67,3 +67,10 @@
+- /torch/distributed/**
+- /torch/nn/parallel/**
+- /test/distributed/**
+- /torch/testing/_internal/distributed/**
```
Do we also want to add `/torch/testing/_internal/distributed` under the label?

wconstab replied:
i don't know? i just copied it from CODEOWNERS.  are you thinking it should be under a separate label, and not relevant for most folks?

wz337 replied:
> i don't know? i just copied it from CODEOWNERS. are you thinking it should be under a separate label, and not relevant for most folks?

This is originally in CODEOWNERS under distributed tests. 
Line 57 https://github.com/pytorch/pytorch/pull/112813/files 
So I am wondering whether we should add this as well, since it is definitely relevant.

wz337 replied:
Oh. I was counting the lines. There were 6 entries, including `/torch/distributed/_composable`. I guess it's ok to leave that one for now?

wconstab replied:
i think the distributed/** takes care of distributed/_composable?

wconstab commented:
@pytorchbot merge

wconstab commented:
@pytorchbot merge -f "unrelated failures.."

wconstab commented:
note: although this is landed, i was puzzled at first that it seems to not take effect on PRs people submit.  I think it's just a matter of needing to rebase new PRs on top of this commit on main, so after a little while this should straighten itself out.

fduwjj commented:
@wconstab how do we know this is working right now?
